filters/data-test.properties

```diff
@@ -48,8 +48,10 @@ environment.build.hibernate.connection.validationQuery=select 1 from INFORMATION
 
 # uPortal server configuration properties
 environment.build.uportal.server=localhost:8080
+environment.build.real.uportal.server=localhost:8080
 environment.build.uportal.protocol=http
 environment.build.uportal.context=/uPortal
+environment.build.real.uportal.context=/uPortal
 environment.build.uportal.secureSessionCookie=false
```

Comment:
Also, setting "real" to the original value would be a good default, as in
`environment.build.real.uportal.server=${environment.build.uportal.server}`

Same for context.